src/Icons/src/Command/SearchIconCommand.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Icons\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Completion\CompletionInput;
+use Symfony\Component\Console\Completion\CompletionSuggestions;
+use Symfony\Component\Console\Cursor;
+use Symfony\Component\Console\Formatter\OutputFormatterStyle;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableCell;
+use Symfony\Component\Console\Helper\TableCellStyle;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\UX\Icons\Iconify;
+
+/**
+ * A console command to search icons and icon sets from iconify.design.
+ *
+ * @author Simon André <smn.andre@gmail.com>
+ *
+ * @internal
```

Comment:
Personal preference of mine: so it's less likely that IDEs intellisense will suggest the class (or if it does, it's struck out in the UI).

Reply:
During the experimental period we don't want to garantee any interface / usage.. So it's a way to say "don't plug this command on anything ... or be ready to see changes" 

The @final is something about code, adding "internal" here is more about the outside usage... (like someone coding something that interact with the command directly from the CLI).

Just a way to say "this can change"